Jenkinsfile

```diff
@@ -157,6 +157,21 @@ pipeline {
             }
         }
 
+        stage('Build ORT Docker image') {
+            agent {
+                dockerfile {
+                    additionalBuildArgs DOCKER_BUILD_ARGS
+                    args DOCKER_RUN_ARGS
+                }
+            }
+
+            steps {
+                sh '''
+                    /opt/ort/bin/ort $LOG_LEVEL --version
+                '''
+            }
+        }
+
```

Comment:
According to the commit message, shouldn't this code be moved from somewhere else?

Reply:
No, this is a new "dummy" stage to build the image before its being used.

Reply:
I've updated the commit message to make that clearer.